### DISCUSSION TITLE
In the vsgQt demo, add the ToolBar to QMaindow

### DISCUSSION THREAD

**Discussion by WdsPjm** [vsg-dev/VulkanSceneGraph#1016] [General]
In vsgQt demo, add ToolBar in QMaindow. If you add Toolbar directly in demo, there will be problems. Make some of the following changes in hopes of helping others。

![image](https://github.com/vsg-dev/VulkanSceneGraph/assets/52599568/f44f9adc-ef1b-46b1-96da-5a0be4cf6124)

![image](https://github.com/vsg-dev/VulkanSceneGraph/assets/52599568/f25392a6-d292-4463-82e9-8ab21eaffe07)

**Comment by robertosfield** [comment]
It's not possible to figure out what you mean with trying to reproduce the issue and trying to spot your changes.  Could you explain what the issue was and provide a diff that illustrates the changes you made. Thanks.

**Comment by WdsPjm** [comment]
The difference is auto window = createWindow(viewer, windowTraits, vsg_scene, nullptr, "First Window");
The fourth parameter to createWindow, previously passed nullptr as the parent.

![image](https://github.com/vsg-dev/VulkanSceneGraph/assets/52599568/701b0b18-17bc-4783-82b1-e26ef9b3114b)

**Reply by robertosfield** [reply]
A git diff is good way of presenting changes, avoids the need for editing screenshots and mostly avoids ambiguity.

I'm a not a Qt expert, why is the (new QWindow) required for the toolbar?

**Reply by WdsPjm** [reply]
I want to use vsg and qt to develop complex software. More than just showing a gltf model.

**Reply by robertosfield** [reply]
That's good to hear, but I still don't know why the (new QWindow) was required to make things work. I'd like to know what Qt isn't working unless you have this.
